client/packages/components/add-package/index.js

```diff
@@ -175,34 +175,32 @@ const AddPackageDialog = ( props ) => {
 					</FormFieldset> )
 				: null
 			}
-			<FormFieldset className="wcc-shipping-add-package-weight-group">
-				<div className="wcc-shipping-add-package-weight">
+			<FormFieldset className="add-package__weight-group">
+				<div className="add-package__weight">
 					<FormLabel htmlFor="box_weight">{ __( 'Package weight' ) }</FormLabel>
 					<FormTextInput
 						id="box_weight"
 						name="box_weight"
 						placeholder={ __( 'Weight of box' ) }
 						value={ box_weight || '' }
-						className={ fieldClassName }
```

Comment:
That additional class name was only added so the `:disabled` styles would be applied. I've moved those styles [here](https://github.com/Automattic/woocommerce-services/pull/1051/files/995d9174509fc081ffa6a5d130b4007088fedcaf#diff-615f3d2f3212d2af594e2cac359817f7R22) instead, and since the `<FormTextInput>` is set to `disabled=true` when the package isn't user-defined, the additional class name is no longer needed.